CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ All notable changes to TTT2 will be documented here. Inspired by [keep a changel
 - Added `SWEP:ClearHUDHelp()` to allow blanking the help text, for dynamically updating help text on equipment (by @EntranceJew)
 - Throwables (grenades) now have a `:GetPullTime()` accessor
 - Throwables (grenades) show UI for the amount of time remaining before detonation (fuse time) (by @EntranceJew)
+- `gameEffects` library for global effects that are useful, such as starting fires (by @EntranceJew)
```

Comment:
Maybe mention that vFire integration is now builtin?

Reply:
not needed

Reply:
I think it is and don't resolve my comments

Reply:
I second this, just add `  - this library has also built-in vFire support meaning that TTT2 now fully supports vFire`

Reply:
![image](https://github.com/TTT-2/TTT2/assets/5711436/5ebb6aba-8260-4cc2-9068-0fdb67015991)
?

Reply:
if there are objections to how vFire itself functions then there's no reason to support it

Reply:
You just removed it wow. That's not remotely what we told you to do, but idc